modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc

```diff
@@ -24,15 +24,20 @@ using custom profiles defined in the `profile:` section of the Tuned CR:
 * usb
 * video
 * vm
+* bootloader
 
 There is some dynamic tuning functionality provided by some of these plug-ins
 that is not supported. The following TuneD plug-ins are currently not supported:
 
-* bootloader
 * script
 * systemd
 
 
+[WARNING]
+====
+This functionality is currently only supported on {op-system-first} 8.x worker nodes. On {op-system-base-full} 7.x worker nodes the TuneD `[bootloader]` plug-in is currently not supported.
```

Comment:
```suggestion
This functionality is currently supported on {op-system-first} 8.x worker nodes. For {op-system-base-full} 7.x worker nodes, the TuneD `[bootloader]` plug-in is currently not supported.
```

Reply:
"This functionality" ? Can you be more specific ?

Reply:
Addressed